corehq/apps/reminders/tests/test_recipient.py

```diff
@@ -65,9 +65,9 @@ def test_recipient_case_owner_location_parent(self):
                 self.assertEqual(reminder.recipient, [parent_location])
 
             # Remove parent location
-            parent_location.delete()
             child_location.parent = None
             child_location.save()
+            parent_location.delete()
```

Comment:
@gcapalbo can you confirm that this fits the intent of the test?  `.delete()` is a cascading delete, so it also deletes the child location.  This would automatically unassign the user from the location, saving them and creating a document update conflict when the user is saved again below.